raiden/transfer/state.py

```diff
@@ -115,11 +115,12 @@ def __init__(
         self.payment_mapping = PaymentMappingState()
 
     def __repr__(self):
-        return '<ChainState block:{} networks:{} qty_transfers:{} chain_id:{}>'.format(
+        return '<ChainState block:{} networks:{} qty_transfers:{} chain_id:{} at {}>'.format(
```

Comment:
why are you adding the id of the object in the representation?

Reply:
1) it's convention
2) it's useful if you need to check identity of the object (i.e. if you by mistake process the same object twice, which I did)

I can remove it ofc if you think the id is redundant...

Reply:
For the classes inside the modules `state`, `state_change`, and `events` I think it's a bit unecessary, these classes have structural and not referential equality.

Reply:
ok, I've reverted the files